maxminddb/__init__.py

```diff
@@ -27,8 +27,11 @@ def open_database(database, mode=MODE_AUTO):
             * MODE_AUTO - tries MODE_MMAP_EXT, MODE_MMAP, MODE_FILE in that
                           order. Default mode.
     """
-    if (mode == MODE_AUTO and maxminddb.extension and
-            hasattr(maxminddb.extension, 'Reader')) or mode == MODE_MMAP_EXT:
+    has_extension = maxminddb.extension and hasattr(maxminddb.extension, 'Reader')
+    if (mode == MODE_AUTO and has_extension) or mode == MODE_MMAP_EXT:
+        if not has_extension:
+            raise ImportError(
```

Comment:
I think I'd expect a `ValueError` if I passed an invalid parameter, not an `ImportError`.